src/components/PageBanner/PageBanner.tsx

```diff
@@ -8,9 +8,11 @@ export interface PageBannerProps {
     imagePath?: string;
 }
 
+const bannerSrc = 'jz20-banner-background.png';
+
 export function PageBanner(props: PageBannerProps) {
     const path = !props.imagePath ? '2019/images/' : props.imagePath;
-    const imagePath = `https://d3o108dy577i1m.cloudfront.net/${path}${props.imageName}.jpg`;
+    const imagePath = props.imageName === bannerSrc ? `/2020/images/${bannerSrc}` :`https://d3o108dy577i1m.cloudfront.net/${path}${props.imageName}.jpg`;
```

Comment:
tanken her var att det skulle blir lättare att flytta ting til cloudfront hvis det hade vart behov for det...

Reply:
gjenom att fjerne den sjekken props.imageName === bannerSrc